### ISSUE TITLE
Issues related to cursor position, including compatibility with stable-cursor tweak

### ISSUE THREAD

**Issue by pronvit** [mifki/df-twbt#23]
Also see http://www.reddit.com/r/dwarffortress/comments/2ohcah/tweak_stablecursor_doesnt_work_in_fullscreen_mode/

**Comment by txtsd** [comment]
DEFINE_VMETHOD_INTERPOSE(void, feed, (std::set<df::interface_key> *input))
in dungeonmode.hpp might be the culprit since the one in stable-cursor.h is the what does the stabilizing.

**Comment by lethosor** [comment]
I don't think that would affect fortress mode, and vmethod hooks shouldn't conflict with each other (at least, they shouldn't disable each other).

**Comment by lethosor** [comment]
I haven't seen any obvious issues with `tweak stable-cursor` on the `next` branch. Can you still reproduce what was happening before?
